src/library/filescanner_ffmpeg.c

```diff
@@ -173,6 +174,28 @@ parse_albumid(struct media_file_info *mfi, const char *id_string)
   return 1;
 }
 
+static int
+parse_rating(struct media_file_info *mfi, const char *rating_string)
+{
+  uint32_t *rating = (uint32_t *) ((char *) mfi + mfi_offsetof(rating));
+
+  if (safe_atou32(rating_string, rating) != 0)
+    return 0;
+
+  // adjust for internal rating 0-100 range redistributing heurisstically
```

Comment:
I don't understand this

Reply:
Some taggers set rating in rnage 0..255 - this code adjusts the file rating to 0..100 that the server uses